samcli/lib/hook/hook_wrapper.py

```diff
@@ -191,7 +191,7 @@ def get_available_hook_packages_ids() -> List[str]:
     LOG.debug("Return available internal hook packages")
     hook_packages_ids = []
     for child in INTERNAL_PACKAGES_ROOT.iterdir():
-        if child.is_dir():
+        if child.is_dir() and child.name[0].isalpha():
```

Comment:
uppercase letter would still pass this check?

maybe something like this?
```suggestion
string.ascii_lowercase

if child.is_dir() and child.name[0] in string.ascii_lowercase:
```

and just to certain, child.name[0] should never throw any error right? (as in is it possible to have an empty string as dir name?

Reply:
I think it is OK .. it does not really matter